packages/mobile/src/app/saga.ts

```diff
@@ -310,10 +327,58 @@ export function* handleSetAppState(action: SetAppState) {
   }
 }
 
+export function* handleSaveOtaTranslations(language: string, translations: Resource) {
```

Comment:
Can you centralize the caching functionality into the same file?  It's spread out across this file and packages/mobile/src/i18n.ts.

Reply:
good idea! i have moved these to the i18n file :)